subxt/src/tx/tx_client.rs

```diff
@@ -2,21 +2,23 @@
 // This file is dual-licensed as Apache-2.0 or GPL-3.0.
 // see LICENSE for license details.
 
-use super::TxPayload;
+use std::borrow::Cow;
+
+use codec::{Compact, Decode, Encode};
+use derivative::Derivative;
+
 use crate::{
     client::{OfflineClientT, OnlineClientT},
     config::{Config, ExtrinsicParams, Hasher},
     error::Error,
     tx::{Signer as SignerT, TxProgress},
     utils::{Encoded, PhantomDataSendSync},
 };
-use codec::{Compact, Encode};
-use derivative::Derivative;
-use std::borrow::Cow;
-
 // This is returned from an API below, so expose it here.
 pub use crate::rpc::types::DryRunResult;
 
+use super::TxPayload;
```

Comment:
I would prefer to move this `use` above the `pub use` as it was before to have clean structure of the use stuff.